src/Watchy.cpp

```diff
@@ -126,6 +127,7 @@ void Watchy::handleButtonPress() {
       }
       showMenu(menuIndex, true);
     } else if (guiState == WATCHFACE_STATE) {
+      button2();
```

Comment:
```suggestion
      handleButton3Press();
```

Same deal, with this one being SW3:

![image](https://user-images.githubusercontent.com/820984/168224736-17f670f0-a198-4629-8ba9-59c45e5acd3f.png)